ocpmodels/common/utils.py

```diff
@@ -1237,9 +1239,13 @@ def update_config(base_config):
                     "magnitude_error",
                 ],
             },
+            "primary_metric": "forces_mae"
         }
         if "primary_metric" in config["task"]:
             _eval_metrics["primary_metric"] = config["task"]["primary_metric"]
+        elif "eval_metrics" in config and "primary_metric" in config["eval_metrics"]:
+            _eval_metrics["primary_metric"] = config["eval_metrics"]["primary_metric"]
```

Comment:
Again, `eval_metrics` shouldn't exist in `config`. Did you include this when trying to use a new checkpoint with the OCPCalculator? If so, https://github.com/Open-Catalyst-Project/ocp/blob/daf72a502c6ee19f2c70eaef58eefc233f25362c/ocpmodels/common/relaxation/ase_utils.py#L151 this actually needs to be updated to only be conditional if it detects an old config. @janiceblue brought up a similar issue.

Reply:
Ok, I copied the logic from base_trainer into the OCP calculator and will see if that works. I reverted the changes I made here